xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
+      "type": "number",
+      "description": "x-ray wavelength"
+    },
+    "bt_safN":{
+      "type": "string",
+      "description": "Safety Approval Number of beamtime"
+    },
+    "bt_experimenters":{
+      "type": "array",
+      "description": "list of experimenter names in [first, last,...] format"
+    },
+    "bt_uid":{
+      "type": "string",
+      "description": "auto-generated uid for Beamtime object"
+    },
+    "sample_name":{
+      "type": "string",
+      "description": "name of sample being measured"
+    },
+    "sample_composition":{
+      "type": "object",
+      "description": "dictionary of sample stoichiometry"
+    },
+    "sample_phase":{
+      "type": "object",
+      "description": "dictionary of sample phase information"
+    },
+    "sample_maker":{
+      "type": "array",
+      "description": "list of sample maker names "
+    },
+    "sa_uid":{
+      "type": "string",
+      "description": "auto-generated uid for Sample object"
+    },
+    "cif_name":{
+      "type": "array",
+      "description": "list of possible structure filenames"
+    },
+    "collaborators":{
+      "type": "array",
+      "description": "list of collaborator names"
+    },
+    "database_id":{
+      "type": "object",
+      "description": "dictionary of structure uid to different database"
+    },
+    "lead_experimenter":{
+      "type": "array",
+      "description": "list of lead experimenter names"
+    },
+    "notes":{
+      "type": "string",
+      "description": "memorable note for sample"
+    },
+    "tags":{
+      "type": "array",
+      "description": "list of short tags user sets up for future search"
+    },
+    "sp_plan_name":{
+      "type": "string",
+      "description": "name of ScanPlan"
+    },
+    "sp_args":{
+      "type": "array",
+      "description": "positional arguments passed to ScanPlan"
+    },
+    "sp_kwargs":{
+      "type": "object",
+      "description": "keyword arguments passed to ScanPlan"
+    },
+    "sp_uid":{
+      "type": "string",
+      "description": "auto-generated uid for ScanPlan object"
+    },
+    "sp_time_per_frame":{
+      "type": "number",
+      "description": "acquisition time per frame in second"
+    },
+    "sp_num_frames":{
+      "type": "number",
+      "description": "number of frames being summed per trigger"
+    },
+    "sp_requested_exposure":{
+      "type": "number",
+      "description": "user requested exposure time in second"
+    },
+    "sp_computed_exposure":{
+      "type": "number",
+      "description": "program computed exposure time"
+    },
+    "sp_type":{
+      "type": "string",
+      "description": "ScanPlan type of xpdacq"
+    },
+    "sp_startingT":{
```

Comment:
This is fine, but I wouldn't go too far in specifying such scan plan specific things. It is fine for the users to inject their own metadata and it is ok if we don't validate that injected metadata.